db/schema.rb

```diff
@@ -126,10 +126,14 @@
   create_table "user_settings", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.boolean  "receive_notification",            default: true
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/StringLiterals
**Severity**: INFO
**File**: db/schema.rb L127

Prefer single-quoted strings when you don&apos;t need string interpolation or special symbols.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*